nucypher/cli/commands/alice.py

```diff
@@ -358,31 +378,27 @@ def derive_policy_pubkey(general_config, label, character_options, config_file):
 
 
 @alice.command()
-@click.option('--bob-encrypting-key', help="Bob's encrypting key as a hexadecimal string", type=click.STRING,
-              required=True)
-@option_bob_verifying_key
-@option_label(required=True)
-@option_m
-@option_n
-@option_rate
-@click.option('--expiration', help="Expiration Datetime of a policy", type=click.STRING)  # TODO: click.DateTime()
-@click.option('--value', help="Total policy value (in Wei)", type=types.WEI)
-@group_character_options
 @option_config_file
 @group_general_config
+@group_character_options
+@option_label(required=True)
+@option_bob_verifying_key
+@click.option('--bob-encrypting-key', help="Bob's encrypting key as a hexadecimal string", type=click.STRING, required=True)
+@click.option('--expiration', help="Expiration Datetime of a policy", type=click.STRING)  # TODO: click.DateTime()
```

Comment:
Should we indicate the expected format of expiration date? - I think it's supposed to be in iso8601